app/javascript/components/common/CopyToClipboardButton.tsx

```diff
@@ -74,7 +74,7 @@ export function CopyToClipboardButton({ textToCopy }: { textToCopy: string }) {
       className={`c-copy-text-to-clipboard ${justCopied ? 'copied' : ''}`}
       aria-label={`Copy ${textToCopy} to the cliboard`}
     >
-      <div className="text">exercism download --exercise=pattern</div>
+      <div className="text">{textToCopy}</div>
```

Comment:
@ErikSchierboom I noticed that this widget doesn't show the correct text to copy.

Reply:
Ah, silly!

Reply:
Ha. I fixed this in https://github.com/exercism/website/pull/573/files too :)